geometry_msgs/PoseStamped tag_camera
geometry_msgs/PoseStamped gripper_base